### DISCUSSION TITLE
Projetos Angular em produção

### DISCUSSION THREAD

**Discussion by waldemirflj** [frontendbr/forum#795] [Perguntas e Respostas]
Boa noite, 

Fiz a proposta para empresa que trabalho para adotarmos um framework para o nosso front-end, minha sugestão foi o Angular, já trabalhei com o mesmo em algumas startups. Porém, a empresa que trabalho atualmente não curte mudanças e estão há muito tempo fazendo da mesma forma.

Vocês saberiam me dizer quais empresas .com já estão utilizando Angular em produção para eu colocar na minha apresentação?

Obrigado!

**Comment by felipefialho** [comment]
Estamos usando aqui no Cubo.

https://cubo.network/

**Comment by loiane** [comment]
Vários times no banco estão usando na área de private (pessoa jurídica), incluindo o time que trabalho. O update pra v5.x vai agora em dezembro pra prod.

Alguns cases famosos (v4.x):
nba.com
royalcaribbean.com/cruises
forbes.com

**Comment by waldemirflj** [comment]
Obrigado galera.

**Comment by fabiomirandaa** [comment]
[Cetelem](https://www.cetelem.com.br) está usando o AngularJs no front.
